nodejs/awsx/apigateway/api.ts

```diff
@@ -376,6 +410,8 @@ interface SwaggerSpec {
     paths: { [path: string]: { [method: string]: SwaggerOperation; }; };
     "x-amazon-apigateway-binary-media-types"?: string[];
     "x-amazon-apigateway-gateway-responses": Record<string, SwaggerGatewayResponse>;
+    "x-amazon-apigateway-request-validators"?: reqvalidation.Validators;
+    "x-amazon-apigateway-request-validator"?: reqvalidation.RequestValidator;
```

Comment:
what's the diff between these two?

Reply:
You have to define the validators (i.e. validate all things, just the body, just the params) & name them. Then "x-amazon-apigateway-request-validator" sets the validator to used at the API level. You can also set a different validator at each method level which will override the API level one.